react/components/Metrics/DataAnalysis/CustomElements/CustomTooltip.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import { InjectedIntlProps, injectIntl } from 'react-intl'
+import { PageBlock } from 'vtex.styleguide'
+
+
+interface Props extends InjectedIntlProps {
+  active?: boolean
+  label?: string
+  payload?: any
+  type?: string
+}
+
+
+class CustomTooltip extends Component<Props> {
+  constructor(props: Props) {
+    super(props)
+  }
+
+  public handleLabel = (label: string) => {
+    // After it will treat intl
+    return label
+  }
+
+  public handleValue = (value: number) => {
+    const { intl } = this.props
+    return intl.formatNumber(value)
+  }
+
+  public render() {
+    const { active } = this.props
+
+    if (active) {
+      const { payload, label } = this.props
+
+      return (
+        <PageBlock>
+          <div className="t-small">
+            <div className="pa2">
+              {label}
+            </div>
+            <div className="flex justify-between" >
+              <div className="pa2" style={{ color: payload[0].color }} >
+                {`${this.handleLabel(payload[0].dataKey)}:  `}
```

Comment:
Actually, I think it is better to maintain in english to info concerning the metric. For example, MemoryUsage has info such as: external, rss, heapUsed and heapTotal. Some of them does not make sense if translated.

But I'll change to use <FormattedNumber> instead of handleValue